plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -442,13 +446,13 @@ private boolean validateRequiredGroups(MissingRequiredProperties validationFailu
     if (missingGroupEntry != null) {
       StringBuilder errorBuilder = new StringBuilder("Missing value for group ");
       errorBuilder.append(missingGroupEntry.getKey());
-      errorBuilder.append(". Properties satisfying group requirement are ");
+      errorBuilder.append(". Properties satisfying group requirement are "); //$NON-NLS-1$
       Set<String> groupMembers = new HashSet<>();
       for (PipelineOptionsProperty missingProperty : missingGroupEntry.getValue()) {
         groupMembers.add(missingProperty.getName());
       }
       errorBuilder.append(MISSING_GROUP_MEMBER_JOINER.join(groupMembers));
-      errorBuilder.append(".");
+      errorBuilder.append("."); //$NON-NLS-1$
       setErrorMessage(errorBuilder.toString());
```

Comment:
I think we can make this work with ICU's `com.ibm.icu.text.ListFormatter`.  ICU is bundled with Eclipse.